### PR TITLE
Added AudioFileWaveform View

### PR DESCRIPTION
**Pull request by Matt54**
Added AudioFileWaveform View

Also added Resources folder to AudioKitUI package, added "drumloop.wav" to resources, and created a TestAudioURLs enum to conveniently get the url to any test audio files we use for visualizations

image
^^Example using url to "drumloop.wav"

Plan was to use this view for: https://github.com/AudioKit/Cookbook/issues/71

### CODE REVIEW

**Comment by aure** [comment]
Woohoo!

File: Sources/AudioKitUI/Visualizations/AudioFileWaveform.swift [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    @ObservedObject var viewModel: AudioFileWaveformViewModel
+
+    public init(url: URL, rmsWindowSize: Double = 256) {
+        viewModel = AudioFileWaveformViewModel(url: url,
+                                               rmsWindowSize: rmsWindowSize)
+    }
+
+    public var body: some View {
+        AudioWaveform(rmsVals: viewModel.rmsValues)
+            .fill(Color.gray)
```
Comment (willsmillie) [inline Sources/AudioKitUI/Visualizations/AudioFileWaveform.swift:50]: Is there a recommended way to override this? Or is it be best for one to write their own implementation? 

Perhaps by adding an argument with a default of `gray`?

either way, thx!

Comment (Matt54) [reply Sources/AudioKitUI/Visualizations/AudioFileWaveform.swift:50]: https://github.com/AudioKit/AudioKitUI/pull/57

Comment (willsmillie) [reply Sources/AudioKitUI/Visualizations/AudioFileWaveform.swift:50]: NOICE! Thank you :)

**Pull request merged**